### DISCUSSION TITLE
Dark Reader for .html files

### DISCUSSION THREAD

**Discussion by Christian-Kouts** [darkreader/darkreader#8488] [Ideas]
Hello, I work with a lot of `.html` files but sadly they are "protected by browser", more specifically "This page is protected by Browser" within the extension. I've done some [reading](https://github.com/Eloston/ungoogled-chromium/issues/1262) and believe this is a choice on Dark Reader's end. I use [Brave Browser](https://brave.com/) and this change would be amazing!

**Comment by Gusted** [comment]
> Hello, I work with a lot of `.html` files but sadly they are "protected by browser", more specifically "This page is protected by Browser" within the extension. I've done some [reading](https://github.com/Eloston/ungoogled-chromium/issues/1262) and believe this is a choice on Dark Reader's end. I use [Brave Browser](https://brave.com/) and this change would be amazing!

Could you give me the full path of such file? Because local files aren't protected by the browsers and aren't currently being specified as protected by dark reader, unless there's some mismatch within the code and detecting the path.

**Reply by Christian-Kouts** [reply]
The files are within [OneDrive](https://en.wikipedia.org/wiki/OneDrive) with file path `C:\Users\chris\OneDrive\Documents`.

**Reply by Gusted** [reply]
Hmm interesting, doesn't seem to be marked as protected on my site, is it marked as "file" by chrome: 
![image](https://user-images.githubusercontent.com/25481501/160298918-2d38fc98-7a41-4a97-bb42-86688beb200f.png)

**Reply by Christian-Kouts** [reply]
Not sure where to find that piece of info but within Chrome I get the same result as in Brave (also built on chromium).

![chrome_onedrive_pdf_darkreader](https://user-images.githubusercontent.com/77142490/160299616-a19df3aa-a623-4e0b-a4a3-91d92703bbd0.jpg)

**Reply by Gusted** [reply]
But PDF != .htmt

**Reply by orionflame** [reply]
I also started seeing this recently. I have the allow accessing local files option turned on but dark reader says the page is protected. How can I fix this?

<img width="921" alt="image" src="https://github.com/user-attachments/assets/c0af50df-0c4f-401a-a567-1c381d86b5d8">
